There is a single none-ASCII character in acountry.c

Is this supposed to be UTF-8 ?

```
.../c-ares-1/acountry.c:276:22: warning: illegal character
      encoding in string literal [-Winvalid-source-encoding]
       { 248, "ax", "<C5>land Island"                         },
                     ^~~~
1 warning generated.
```

--- a/acountry.c
+++ b/acountry.c
@@ -273,7 +273,7 @@ static const struct search_list *list_lookup(int number, const struct search_lis
  */
 static const struct search_list country_list[] = {
        {   4, "af", "Afghanistan"                          },
-       { 248, "ax", "Åland Island"                         },
+       { 248, "ax", "Ã…land Island"                         },
        {   8, "al", "Albania"                              },
        {  12, "dz", "Algeria"                              },
        {  16, "as", "American Samoa"                       },

